bot/resources/snakes/snake_facts.json

```
@@ -3,7 +3,7 @@
         "fact": "The decapitated head of a dead snake can still bite, even hours after death. These types of bites usually contain huge amounts of venom."
     },
     {
-        "fact": "What is considered the most “dangerous” snake depends on both a specific country’s health care and the availability of antivenom following a bite. Based on these criteria, the most dangerous snake in the world is the saw-scaled viper, which bites and kills more people each year than any other snake."
+        "fact": "What is considered the most 'dangerous' snake depends on both a specific country’s health care and the availability of antivenom following a bite. Based on these criteria, the most dangerous snake in the world is the saw-scaled viper, which bites and kills more people each year than any other snake."
     },
     {
         "fact": "Snakes live on everywhere on Earth except Ireland, Iceland, New Zealand, and the North and South Poles.a"
```

This one looks like it has a grammatical error as well as a typo.

```suggestion
        "fact": "Snakes live everywhere on Earth except Ireland, Iceland, New Zealand, and the North and South Poles."
```